src/script/page/RightSidebar/GuestServicesOptions/GuestServicesOptions.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {WebAppEvents} from '@wireapp/webapp-events';
+import {amplify} from 'amplify';
+import {FC, useState} from 'react';
+
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+
+import PanelHeader from '../PanelHeader';
+
+import GuestOptions from './components/GuestOptions';
+import ServicesOptions from './components/ServicesOptions';
+
+import {toggleFeature} from '../../../conversation/ConversationAccessPermission';
+import {ConversationRepository} from '../../../conversation/ConversationRepository';
+import {Conversation} from '../../../entity/Conversation';
+import {TeamRepository} from '../../../team/TeamRepository';
+import {TeamState} from '../../../team/TeamState';
+import {ModalsViewModel} from '../../../view_model/ModalsViewModel';
+
+interface GuestServicesOptionsProps {
+  activeConversation: Conversation;
+  conversationRepository: ConversationRepository;
+  teamRepository: TeamRepository;
+  onBack: () => void;
+  onClose: () => void;
+  teamState: TeamState;
+  isGuest?: boolean;
+}
+
+// STATE.GUEST_OPTIONS
+const GuestServicesOptions: FC<GuestServicesOptionsProps> = ({
+  activeConversation,
+  conversationRepository,
+  teamRepository,
+  onBack,
+  onClose,
+  teamState,
+  isGuest = false,
+}) => {
+  // TODO: It's needed here?
+  // getLogger('GuestsAndServicesViewModel')
```

Comment:
I think we can safely remove the `loggers` from new components. 
In the future we will try to have a less verbose more useful logging, but that can be done later
```suggestion
```